script.js

```diff
@@ -1 +1,44 @@
-// this is script file
+// all the dom elements are selected
+const addTodoInputElement = document.getElementById("addTodoInputField");
+const addTodoButton = document.getElementById("addTodoButton");
+const allTodoContainer = document.getElementById("todosContainer");
```

Comment:
To indicate a variable might contain a HTML Element either prefix it with `$`  or postfix with **element**.
```suggestion
const $todoInput = document.getElementById("addTodoInputField");
const $addButton = document.getElementById("addTodoButton");
const $todoList = document.getElementById("todosContainer");
```